### PR TITLE
reset files

### PR DESCRIPTION
**Pull request by AdrianC113**
# Submitting Week 2 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- ✅ Career Readiness
- ✅ Coding Challenges
- ✅ Coding Critique
- ✅ Coding Concepts
- ✅ Interview Questions

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+// // input: an object's number
+// // output: a string that states the object's number and if it is divisible by three or not
+// const divisibleByThree = (object) => {
+//     if (Object.values(object) % 3 === 0) {
+//         // iterate using modulo operator and see if each value in the object is divisible by three
+//        return `${Object.values(object)} is divisible by three`
+//        // return string interpolation calling forth the object's value along with a note of whether it is divisible by three or not
+//     } else {
+//         return `${Object.values(object)} is not divisible by three`
```
Comment (chrisyhlee) [inline code-challenges.test.js:77]: The modulo operator does not start iteration. For the given variables, there is only 1 key value pair in each object. Also, it's important to note that `Object.values()` will return an array of all of the values in the object, not just the values themselves.

File: code-challenges.test.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+// // Pseudo code:
+
+// // input: array of strings
+// // output: array with all strings with the first letter capitalized
+// const capitalizedWords = (arr) => {
+//     // iterate using .map
+//     return arr.map((value) => {
+//         // return using .toUpperCase to capitalize the letter under the 0 index and .slice to insert the rest of the string from the 1 index and so forth
+//         return value[0].toUpperCase() + value.slice(1)
```
Comment (chrisyhlee) [inline code-challenges.test.js:108]: I would study up on vocabulary. A better way to say this pseudo code on line 107 would be:
- each iteration capitalizes the character at index 0, then concatenates the rest of the string to the capitalized letter.

File: code-critique.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+// Compared to the python code shown above, the javascript code was a bit longer and takes a few more steps to accomplish the goal. 
+
+// Are there best practices that are not being followed? 
+// I didnt think about "return -1" but I later realized that it made sense when dealing with index that start from 0 and so on.
+
+// Would you take a different approach? 
+// I may have thought about using a Higher Order Function like .split and .map to try and iterate every character of the string to achieve similar results.
+
+// What did you learn? 
+// There are many ways to achieve the same goal, whether it is a different language or a different method within each language.
```
Comment (chrisyhlee) [inline code-critique.js:70]: It's important to be able to decipher what is good vs bad in a response given from AI. The prompt said to log the index of the first vowel, so why did AI give a string instead? It also logged the string, and then returned just the index on the next line, so which one is it trying to do?
The takeaway is to make sure we are using AI to benefit our understanding of concepts and learn, not just to take the response it gives as gospel.

**Review by chrisyhlee — APPROVED** [review]
I see you're trying to use new tools, which is great, just make sure you fully understand what it is and what it does. I left some comments for you to read through. My main suggestion is to make sure you are explaining things accurately in your pseudo code. For example, you stated that you are using the modulo operator to iterate, but the modulo operator does not actually start iteration. Little things like this have a big impact on how your understanding of the concepts look to others, such as interviewers. Overall this assessment displays Novice, I think you're doing a great job!

**Comment by AdrianC113** [comment]
Thanks Chris! will definitely focus more on the vocab. I do have some slight trouble pseudo coding and navigating because of that. Also, will for sure not take Chat GPT or other AI's outputs as gospel, great point! I just took it and ran with it, should have focused more on the requirement while reading the output to see if it was giving out what was required. Thanks again for looking over my assessment Chris!

**Pull request merged**